internal/metrics.go

```diff
@@ -33,12 +36,28 @@ func newMetrics() *metrics {
 		Name: "fleetlock_unlock_request_count",
 		Help: "Number of unlock requests",
 	})
+	totalRequests := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "http_requests_total",
+		Help: "Number of requests",
+	}, []string{"path"})
+	responseStatus := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "http_response_status",
+		Help: "Status of HTTP response",
+	}, []string{"path", "status"})
+	httpDuration := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "http_response_time_millis",
```

Comment:
https://pkg.go.dev/github.com/dnesting/client_golang/prometheus#example-InstrumentHandler http_request_duration_microseconds or http_request_duration_milliseconds

Reply:
The instrumented handler doesn't actually track everything I want to track and tracks some stuff I don't care about. Rather than try to do both I think it's okay to just use this custom one.